libs/ui/elements/src/lib/record-api-form/record-api-form.component.html

```diff
@@ -28,7 +28,7 @@
             <input
               class="mr-2 cursor-pointer"
               type="checkbox"
-              [checked]="(limit$ | async) === '-1'"
+              [checked]="(limit$ | async) === '-1' || (limit$ | async) === ''"
```

Comment:
I think we could introduce two Observables for the inputs that use logic in this template. This would prevent multiple subscriptions for the same logic and multiplication of the same expression if we need to override the template.